ppg-server/ppg-server.spec

```diff
@@ -0,0 +1,30 @@
+%global sname   percona-ppg-server
+%global pgmajorversion 14
+%global version 4
+
+Summary:        Base selection of PostgreSQL%{pgmajorversion} components
+Name:           %{sname}-%{pgmajorversion}
+Version:        %{version}
+Release:        1%{?dist}
+Epoch:          1
+License:        PostgreSQL
+Group:          Applications/Databases
+URL:            https://www.percona.com/software/postgresql-distribution
+Packager:       Percona Development Team <https://jira.percona.com>
+Vendor:         Percona, LLC
+BuildArch:      noarch
+
+Requires:       percona-postgresql%{pgmajorversion}-server
+Requires:       percona-postgresql-common
+Requires:       percona-pg-stat-monitor%{pgmajorversion}
+Requires:       percona-pgaudit
+Requires:       percona-pg_repack%{pgmajorversion}
+Requires:       percona-postgresql%{pgmajorversion}-contrib
+Requires:       percona-wal2json%{pgmajorversion}
+
+%description
+Percona Distribution for PostgreSQL provides the best and most critical
+enterprise components from the open-source community, in a single distribution,
+designed and tested to work together.
+
+%files
```

Comment:
Also specify change log.

Reply:
@EvgeniyPatlan What should I state there? Just the changelog entry and that I've created the file initially?

Reply:
yeah I usually add 'Initial build'

Reply:
Added it now